sdk/formrecognizer/ai-form-recognizer/package.json

```diff
@@ -112,6 +108,7 @@
     "eslint-plugin-no-null": "^1.0.2",
     "eslint-plugin-no-only-tests": "^2.3.0",
     "eslint-plugin-promise": "^4.1.1",
+    "fs-extra": "^8.1.0",
```

Comment:
what's actually using fs-extra? I didn't see it in the base rollup config except as an external

Reply:
good question. investigating why I did this.

Reply:
We are using it to create file streams for testing in Node.